package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "leaderboard",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "babel,config.js",
   "scripts": {
     "build": "webpack --mode production",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "start": "webpack",
+    "dev": "webpack serve"
```

Comment:
- [ ] As the previous code reviewer requested, some scripts are not running properly, like the `npm start`. Kindly change the script so it can run following the suggested [documentation](https://webpack.js.org/guides/getting-started/). See this example:  
 -  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }